src/bar.cpp

```diff
@@ -1,7 +1,7 @@
 #include "bar.hpp"
-#include <spdlog/spdlog.h>
 #include "client.hpp"
 #include "factory.hpp"
+#include <spdlog/spdlog.h>
```

Comment:
Given spdlog itself bundles fmtlib we should probably pass `-DSPDLOG_FMT_EXTERNAL` from meson.build instead to keep fmt API usage consistent.

Reply:
Yeah we should use `SPDLOG_FMT_EXTERNAL`

Reply:
Will do it in another PR